md/templates/md/agency_detail.html

```diff
@@ -1,12 +1,86 @@
 {% extends "agency_detail.html" %}
 
 {% block census-display %}{% endblock census-display %}
-{% block stop-display %}{% endblock stop-display %}
+
+{% block stop-display %}
+<!-- stop data -->
+<div class="row">
+  <div class="col-md-12">
+    <h2 id="stop-percentage">Stops data</h2>
+  </div>
+
+  <div class="col-md-4">
+    <h3>Stops by ethnicity</h3>
+
+    <div>
+      <svg id="stop_race_pie"></svg>
+    </div>
+
+    <p class="help-block">
+      This graph reflects the racial composition of drivers stopped by law
+      enforcement officers in the jurisdiction since the department began
+      reporting its data to the NC Department of Justice. Adjusting the drop
+      down menu will display the racial breakdown of stops on a year-by-year
```

Comment:
Does the client know that this is one of the text blocks they should provide and/or should we go ahead and make a reasonable attempt to fix this?  (I guess "NC Department of Justice" should be "Maryland XXX" (ask Colin) and "racial" should be "ethnic".)